editors/code/package.json

```diff
@@ -199,6 +199,11 @@
                 "title": "Reload workspace",
                 "category": "rust-analyzer"
             },
+            {
+                "command": "rust-analyzer.addProject",
+                "title": "Add current file to workspace",
```

Comment:
Maybe something like `Add project of current file to workspace`? (quite annoying how workspace has so many different meanings with cargo and vscode involved

Reply:
That can work. I'll think of some other options (if I can) and get back to you on this in a few hours.

Reply:
i've renamed it to "Add current file's crate to workspace", which is a bit more precise.